index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="main-container">
+        <div class="nav-bars">
+            <div class="logo-show">
+              <h1>My Logo</h1>
+            </div>
+            <div class="menu">
+              <div><img src="photo/navs/Union.png" alt=""></div>
```

Comment:
Great job @osbentech 🤝

- [ ] [Optional] For Semantic reasons, it would be better if you put this section in a `<nav>` tag instead of a `<div>`tag.